aya/src/programs/mod.rs

```diff
@@ -977,17 +977,7 @@ impl ProgramInfo {
 
     /// The name of the program as was provided when it was load. This is limited to 16 bytes
     pub fn name(&self) -> &[u8] {
```

Comment:
This is probably one for Alessandro, but why are we returning &[u8] instead of `std::ffi::Cstr`

Reply:
I think that's because the string is not null terminated if it is exactly 16 bytes long.

Reply:
Is that true? from what I've read the string is always null terminated

https://stackoverflow.com/questions/47288881/are-c-constant-character-strings-always-null-terminated